src/components/Row/type.ts

```diff
@@ -0,0 +1,22 @@
+import {Prop} from 'vue';
```

Comment:
What is the reason again to put these in different files? I think the naming is a little off. Since these files sometimes also include "normal" variables.